lib/Upcoming.dart

```diff
@@ -1,94 +1,84 @@
+                      child: Text(DateFormat('MMM').format(widget.ride.startTime).toUpperCase() +
+                          ' ' + ordinal(int.parse(DateFormat('d').format(widget.ride.startTime))) +
+                          ' ' + DateFormat('jm').format(widget.ride.startTime),
+                          style: TextStyle(
+                            color: Colors.black,
+                            fontSize: 30,
+                            fontFamily: 'SFProDisplay',
+                            fontWeight: FontWeight.bold,
+                          )
+                      ),
+                    ),
+
                     Container(
-                      width: MediaQuery.of(context).size.width,
-                      child: Row(
-                        children: <Widget>[
-                          SizedBox(width: 20),
-                          TimeLine(),
-                          SizedBox(width: 20),
-                          Expanded(
-                            child: Container(
-                              child: Column(
-                                children: <Widget>[
-                                  InformationRow(
-                                      loc: 'Upson Hall',
-                                      address: '124 Hoy Rd, Ithaca, NY 14850',
-                                      action: 'Pickup Passenger 1',
-                                      time: '3:15 PM'),
-                                  SizedBox(height: 20),
-                                  InformationRow(
-                                      loc: 'Gates Hall',
-                                      address: '107 Hoy Rd, Ithaca, NY 14853',
-                                      action: 'Pickup Passenger 2',
-                                      time: '3:20 PM'),
-                                  SizedBox(height: 20),
-                                  InformationRow(
-                                      loc: 'Statler Hall',
-                                      address: '109 Tower Rd, Ithaca, NY 14850',
-                                      action: 'Dropoff Passenger 1',
-                                      time: '3:25 PM'),
-                                  SizedBox(height: 20),
-                                  InformationRow(
-                                      loc: 'Rhodes Hall',
-                                      address: '109 Tower Rd, Ithaca, NY 14850',
-                                      action: 'Dropoff Passenger 2',
-                                      time: '4:20 PM'),
-                                ],
-                              ),
-                            ),
+                      color: Colors.white,
+                      child: Column(
+                        children: [
+                          SizedBox(height: 32),
+                          Contact(),
+                          SizedBox(height: 60),
+                          TimeLine(widget.ride),
```

Comment:
One question I have about the timeline (because I'm not sure how to implement this): Have you thought about if there a way for the timeline to rotate to a new circle based on the status of the ride?

Reply:
This is a good point, I didn't actually see the Current Ride page design--since my page is just used for the upcoming rides right now, I think I'll keep it like this and add that in a PR for Current Ride. I think here's the best way to do it:
- create a key and position getter for each of the 4 rows (on the way, driver arrived, ride card, arrived) so that we can draw lines between each of them and change those colors as needed rather than having just one long line (what I'm doing now)
- change the ride card to take a bool that makes the text bold or not
- change the locationCircle to take in a bool that makes it filled or not
- add a carCircle for the little car icon indicating the current status
- change TimeLineRow to take a widget as the bullet point so that we can pass locationCircle or carCircle as needed